### ISSUE TITLE
Some URLs in fresh/updated Doxyfile are split over two lines

### ISSUE THREAD

**Issue by jwakely** [doxygen/doxygen#8093]
**Describe the bug**

After using `doxygen -u` to update an old Doxyfile to the 1.8.20 format I see some URLs are incorrectly split over two lines:

```
# The QHP_VIRTUAL_FOLDER tag specifies the namespace to use when generating Qt
# Help Project output. For more information please see Qt Help Project / Virtual
# Folders (see: https://doc.qt.io/archives/qt-4.8/qthelpproject.html#virtual-
# folders).
# The default value is: doc.
```
the anchor should be `#virtual-folders` but the second half of it is on a new line after a `#` comment character.

**Expected behavior**

It should be:

```
# The QHP_VIRTUAL_FOLDER tag specifies the namespace to use when generating Qt
# Help Project output. For more information please see Qt Help Project / Virtual
# Folders (see: https://doc.qt.io/archives/qt-4.8/qthelpproject.html#virtual-folders).
# The default value is: doc.
```

**To Reproduce**

`doxygen -g`

**Version**
1.8.20

**Comment by albert-github** [comment]
Looks like this problem is present for a while. In the version 1.8.5 I see it for the first time and it probably due to "Configuration data for the config file, the documentation, and the wizard are now produced from a single source" and has not been considered.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8097

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
